pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1420,6 +1424,20 @@
     |=  [=wire error=(unit tang)]
     ^+  event-core
     ::
+    ?:  ?=([%alien @ ~] wire)
+      ::  if we haven't received an attestation, ask again
+      ::
+      ?^  error
+        %-  (slog leaf+"ames: attestation timer failed: {<u.error>}" ~)
+        event-core
+      ?~  ship=`(unit @p)`(slaw %p i.t.wire)
+        %-  (slog leaf+"ames: got timer for strange wire: {<wire>}" ~)
+        event-core
+      =/  state=(unit peer-state)  (get-peer-state u.ship)
+      ?:  ?=([~ %known *] ship-state)
```

Comment:
ship-state -> state

Reply:
huh. I can see why this didn't trigger a compilation error, but it'd be nice if it could (either by flagging `state` as an "unused face" or the comparison as always false).

Anyway, `=/  ship-state  (~(get by peers.ames-state) u.ship)` seems like the more common pattern in ames, so I went with that.